src/bin/vip-wp.js

```diff
@@ -53,11 +53,27 @@ const unpipeStreamsFromProcess = ( { stdin, stdout: outStream } ) => {
 };
 
 const bindStreamEvents = ( { subShellRl, commonTrackingParams, isSubShell, stdoutStream } ) => {
+	const criticalErrors = [
+		'ECONNRESET',
+		'ETIMEDOUT',
+		'EHOSTUNREACH',
+		'ENOSPC',
+		'EACCES',
+		'EMFILE',
+		'ENOMEM',
+	];
+
 	stdoutStream.on( 'error', err => {
 		commandRunning = false;
 
-		// TODO handle this better
-		console.error( 'Error: ' + err.message );
+		setImmediate( () => {
```

Comment:
We probably don't want to use `setImmediate` as this is a non-standard js function.
https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate

Reply:
You're totally right. Addressed in [e449cc9](https://github.com/Automattic/vip-cli/pull/2092/commits/e449cc91dc519e4f4c315d8da9958366d4360aa2).